README.md

```diff
@@ -49,23 +49,25 @@ The sample library specifies to use `gzip` compression by adding the `Accept-Enc
 
 ## Configure the Sample
 
-Included in the sample is a configuration file with placeholders that need to be replaced with the proper values. They include information for authentication, connecting to ADH, and pointing to a namespace.
+Included in the sample is a configuration file with placeholders that need to be replaced with the proper values. They include information for authentication, connecting to the SDS Service, and pointing to a namespace.
 
-### AVEVA Data Hub
+### Aveva Data Hub
```

Comment:
I've seen a little different in terms of casing for this, but ocs-docs has the company name all upper-case, do you if we are supposed to use this casing instead?

Reply:
yea that was not intentional, it should be all caps. Thanks for pointing that out